tools/configurations/src/config.ts

```diff
@@ -25,6 +25,8 @@ export interface BaseOptions {
   hideLogs?: boolean;
   /** Prevents plugin from executing */
   skip?: boolean;
+  /** Toggle the use of `useImperativeHandle` which alters the `ref` so we can access methods from the custom element. */
+  withImperativeHandle?: boolean;
```

Comment:
Why would you want to diable this?

Reply:
Mostly because I didn't want to break the current use of `useImperativeHandle`. If we could remove `useImperativeHandle` that would make this rather odd option redundant.